File: CarServiceBackend/CarServiceBackend/Models/Booking.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;
using System.ComponentModel.DataAnnotations.Schema;


namespace CarServiceBackend.Models
{
    public class Booking
    {
        [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
        public int BookingId { get; set; }
        public DateTime BookingDate { get; set; }
        public string CarModel { get; set; }
        public string CarProblem { get; set; }
        public bool Approved { get; set; }
        public bool Working { get; set; }
        public bool Ready { get; set; }
        public int UserId { get; set; }
        public User User { get; set; }


    }
}

File: CarServiceFront/src/app/models/booking.ts
export class Booking {
  BookingId:number;
  BookingDate:string;
  CarModel: string;
  CarProblem: string;
  Approved: boolean;
  Working: boolean;
  Ready:boolean;
  UserId: number

  constructor(bDate:string,bCarModel:string, bCarProb: string, bA:boolean, bW:boolean, bR:boolean, bU: number) {
    this.BookingDate = bDate;
    this.CarModel = bCarModel;
    this.CarProblem = bCarProb;
    this.Approved = bA;
    this.Working = bW;
    this.Ready = bR;
    this.UserId = bU;
  }

}
